doc/api/util.md

```diff
@@ -1392,6 +1392,9 @@ added: v10.0.0
 * Returns: {boolean}
 
 Returns `true` if the value is a native `External` value.
+`native external` is data that is:
+i) not stored inside JS heap
+ii) not conformant to JS types
```

Comment:
Suggestion... something like the following would be better here:

```
A native `External` value is a special type of object whose
data is (a) not stored within the JavaScript managed heap
and does not conform to standard JavaScript types. Such
objects are created either by Node.js internals or native
addons and are wrapped by a JavaScript object.
```

Reply:
@jasnell , Thanks. Fixed it.